.github/workflows/webpack.yml

```diff
@@ -3,8 +3,6 @@ name: Build and Deploy
 on:
   push:
     branches: [ "main" ]
```

Comment:
no PRs?

Reply:
it was in there before, but i thought i would simplify it, and took out PR because i'm imagining that this will only run when the PR is merged, and not when a PR is created or updated. i'm happy to add it back in if you think it's needed though!

Reply:
ok, that's totally fine. i just thought we were relying on PRs for content updates. the [instructions](https://github.com/fabric-testbed/knit-website?tab=readme-ov-file#content-management) should probably be updated to match in that case

Reply:
i've made a note in the [issues](https://github.com/fabric-testbed/knit-website/issues/31) to review the content management instructions!